composer.json

```diff
@@ -10,12 +10,16 @@
         "consolidation/robo": "^1.4",
         "gitonomy/gitlib": "^1.0",
         "nuvoleweb/robo-config": "^0.2.1",
-        "jakeasmith/http_build_url": "^1.0.1"
+        "jakeasmith/http_build_url": "^1.0.1",
+        "symfony/yaml": ">3.1.0"
     },
     "require-dev": {
         "openeuropa/code-review": "~1.0.0-beta3",
         "phpunit/phpunit": "~5.5||~6.0"
     },
+    "_readme": [
+        "We need require a version of 'symfony/yaml' higher then '3.1' since versions before 3.1.0 don't have dumper 'DUMP' constants available and 3.1.0 breaks permissions setup in tests."
+    ],
```

Comment:
That's not needed, it's only needed when we fix dependencies when they break builds etc. not when we need a specific version because we use its API, for that having it in the requirements is enough.

Reply:
For me there are two kind of dependencies. The first is when we need the package and it wouldn't be fetched if out of composer requirements. Second kind is like here when we're really requiring the package since it was already being fetched, we're only adding this to the list to adjust the accepted versions. For this later kind of dependencies I think it's positive that we have a reason to understand why it was needed since sometime soon it can be removed, when other packages update their requirements.

Just saying, it's not mandatory, I can remove it.